chromium_src/BUILD.gn

```diff
@@ -218,6 +218,10 @@ static_library("chrome") {
     public_deps += [ "//chrome/services/util_win:lib" ]
   }
 
+  if (is_mac) {
+    public_deps += [ ":chrome_lib_arc" ]
```

Comment:
Needs to be public_deps instead of deps so electron:lib can see the headers: https://gn.googlesource.com/gn/+/master/docs/reference.md#var_public_deps

Reply:
I like the changes in this PR, an additional step to separate would be to make `chrome_lib_arc` a dependency of `chrome` and not `electron_lib`. We build `chrome` as a static library and this new source set can be built as part of it. Any reasons to make it a dependency of `electron_lib` ?